bokehjs/src/lib/document/document.ts

```diff
@@ -662,7 +662,7 @@ export class Document {
     }
   }
 
-  apply_json_patch(patch: Patch, buffers: [any, any][] = [], setter_id?: string): void {
+  apply_json_patch(patch: Patch, buffers: Buffers = new Map(), setter_id?: string): void {
```

Comment:
@philippjfr, so far I didn't find any our code (panel, jupyter_bokeh, ipytwidgets_bokeh) that would break, but I can definitively support the original type along the new one, at least in 2.x branch.

Reply:
Yeah don't expect any disruption from this. Still might make sense to make it backward compatible for the 2.1 release.